service/arm_templates/virtual_network_setup.json

```diff
@@ -69,7 +86,25 @@
                 "id": "[parameters('workerSecurityGroupID')]"
               }
             }
-          }
+          },
+          {
+            "name": "[variables('apiLoadBalancerSubnetName')]",
+            "properties": {
+              "addressPrefix": "[parameters('apiLoadBalancerSubnetCidr')]"
+            }
+          },
+          {
+            "name": "[variables('etcdLoadBalancerSubnetName')]",
+            "properties": {
+              "addressPrefix": "[parameters('etcdLoadBalancerSubnetCidr')]"
+            }
+          },
+          {
+            "name": "[variables('ingressLoadBalancerSubnetName')]",
+            "properties": {
+              "addressPrefix": "[parameters('ingressLoadBalancerSubnetCidr')]"
+            }
+          },
```

Comment:
Do we need virtual network subnets for ingress and API? They have just static public IP. This is where my network foo ends :P

Reply:
You are right, I'll remove them! I'll remove them from the azuretpr too, unless you see something useful we could do with them